intl/msg_hash_us.h

```diff
@@ -8299,18 +8527,34 @@ MSG_HASH(
    MENU_ENUM_SUBLABEL_VIDEO_SHADER_NUM_PASSES,
    "Increase or decrease the amount of shader pipeline passes. Separate shaders can be bound to each pipeline pass and configure its scale and filtering."
    )
+MSG_HASH(
+   MENU_ENUM_LABEL_HELP_VIDEO_SHADER_NUM_PASSES,
+   "RetroArch allows you to mix and match various shaders with arbitrary shader passes, with custom hardware filters and scale factors.\nThis option specifies the number of shader passes to use. If you set this to 0, and use Apply Shader Changes, you use a 'blank' shader.\nThe Default Filter option will affect the stretching filter."
+   )
 MSG_HASH(
    MENU_ENUM_LABEL_VALUE_SHADER,
    "Shader"
    )
+MSG_HASH(
+   MENU_ENUM_LABEL_HELP_VIDEO_SHADER_PASS,
+   "Path to shader. All shaders must be of the same type (i.e. CG, GLSL or HLSL).\nSet Shader Directory to set where the browser starts to look for shaders."
+   )
 MSG_HASH(
    MENU_ENUM_LABEL_VALUE_FILTER,
    "Filter"
    )
+MSG_HASH(
+   MENU_ENUM_LABEL_HELP_VIDEO_SHADER_FILTER_PASS,
+   "Hardware filter for this pass. If 'Don't Care' is set, 'Default Filter' will be used."
```

Comment:
Possible bad inheritance here: at least in Crowdin there is no such option as "Default Filter", only "Default". "Filter" uppercasing and quotation marks should be corrected after confirming it does not exist indeed.

Reply:
Noted, will fix in phase 2 PR.
As a non-native speaker: what is the quotation convention for the US English text? Single quotes (') were used in some places, but it can be unified to have the double quote (").

Reply:
The US convention is to use double quotation marks; but programmers do like to use single quotes, as those do not need escaping in C strings:
"'" - just works
"\\"" - is a little annoying